native/cocos/platform/interfaces/modules/canvas/ICanvasRenderingContext2D.h

```diff
@@ -77,16 +71,16 @@ class CC_DLL ICanvasRenderingContext2D : public OSInterface {
         virtual void            clearRect(float /*x*/, float /*y*/, float w, float h)                                                           = 0;
         virtual void            fill()                                                                                                          = 0;
         virtual void            rect(float x, float y, float w, float h)                                                                        = 0;
-        virtual void            setLineCap(const ccstd::string &lineCap)                                                                        = 0;
-        virtual void            setLineJoin(const ccstd::string &lineCap)                                                                       = 0;
+        virtual void            setLineCap(const std::string &lineCap)                                                                          = 0;
```

Comment:
Why change ccstd::string to std::string?

Reply:
I made a mistake. There was a conflict when merging 3.5